ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallViewOptions.kt

```diff
@@ -5,25 +5,32 @@ import com.revenuecat.purchases.Offering
 class PaywallViewOptions(builder: Builder) {
 
     val offering: Offering?
+    internal val offeringId: String?
```

Comment:
How come this is needed if we already have `Offering`?

Reply:
We can't pass an `Offering` object to the new activity since it's not serializable. As a fallback, in order to display a specific offering in the activity, we allow to pass an offering id and load the offering again from the paywall view itself.

I'm not exposing that API publicly right now, so devs still need to pass the full `Offering` object normally, this would only be used for activities

Reply:
This is a weird definition of the options then.
What do you think about turning this into an enum:
```kotlin
sealed class Selection {
    data class Offering(val offering: Offering) : Selection()
    data class OfferingIdentifier(val id: String) : Selection()
    object None : Selection()
}
```

Reply:
Hmm well, I'm currently not exposing the option to pass the identifier to the paywallView publicly. I guess we could hold that type internally to decide what to do though, since they are indeed exclusive. Will do that!